src/components/svgs/nodes/master.tsx

```diff
@@ -0,0 +1,304 @@
+                    </Button>
+                </>
+            ),
+            oneLine: true,
+        },
+    ];
+
+    const componentField: RmgFieldsField[] = attrs.components.map((c, i) => {
+        const { label, type, defaultValue, value } = c;
+        if (type === 'number' || type === 'text') {
+            return {
+                label: t(label),
+                type: 'input',
+                value: value ?? defaultValue,
+                onChange: v => {
+                    attrs.components[i].value = v;
+                    handleAttrsUpdate(id, { ...attrs, components: attrs.components });
+                },
+            };
+        } else if (type === 'switch') {
+            return {
+                label: t(label),
+                type: 'switch',
+                isChecked: (value ?? defaultValue) === 'true',
+                onChange: v => {
+                    attrs.components[i].value = v;
+                    handleAttrsUpdate(id, { ...attrs, components: attrs.components });
+                },
+            };
+        } else if (type === 'textarea') {
+            return {
+                label: t(label),
+                type: 'textarea',
+                value: value ?? defaultValue,
+                onChange: v => {
+                    attrs.components[i].value = v;
+                    handleAttrsUpdate(id, { ...attrs, components: attrs.components });
+                },
+            };
+        } else {
+            return {
+                type: 'input',
+                label: 'undefined',
+                value: 'none',
+            };
+        }
+    });
+
+    const [isThemeRequested, setIsThemeRequested] = React.useState(false);
+    React.useEffect(() => {
+        if (attrs.color && isThemeRequested && output) {
+            attrs.color.value = output;
+            handleAttrsUpdate(id, { ...attrs, color: attrs.color });
+            setIsThemeRequested(false);
+        }
+    }, [output?.toString()]);
+
+    const colorField: RmgFieldsField[] = [
+        {
+            type: 'custom',
+            label: t('color'),
+            component: (
+                <ThemeButton
+                    theme={attrs.color?.value}
+                    onClick={() => {
+                        setIsThemeRequested(true);
+                        dispatch(openPaletteAppClip(attrs.color?.value));
+                    }}
+                />
+            ),
+        },
+    ];
+
+    return (
+        <>
+            <RmgFields fields={field} />
+            {attrs.randomId !== 'undefined' && <RmgFields fields={componentField} minW="full" />}
+            {attrs.randomId !== 'undefined' && attrs.color !== undefined && (
+                <RmgFields fields={colorField} minW="full" />
+            )}
+            <MasterImport isOpen={openImport} onClose={() => setOpenImport(false)} onSubmit={handleImportParam} />
+            <MasterManager isOpen={openManager} onClose={() => setOpenManager(false)} />
```

Comment:
Is it possible to have this manager in settings instead of the specific attributes? This manages all master nodes not any specific one.